apps/web/src/pages/templates/editor/DigestWorkflowTourTooltip.tsx

```diff
@@ -0,0 +1,126 @@
+import { UnstyledButton } from '@mantine/core';
+import styled from '@emotion/styled';
+import { TooltipRenderProps } from 'react-joyride';
+import { useFormContext } from 'react-hook-form';
+import { StepTypeEnum } from '@novu/shared';
+
+import { useTour } from './TourProvider';
+import { Button, colors, DotsNavigation } from '../../../design-system';
+import { Clock, LetterOpened, BellWithNotification } from '../../../design-system/icons';
+import { useTemplateEditorContext } from './TemplateEditorProvider';
+import { IForm } from '../components/formTypes';
+
+const ICONS = [Clock, LetterOpened, BellWithNotification];
+const TITLE = ['Set-up time interval', 'Set-up email content', 'Test your workflow'];
+const DESCRIPTION = [
```

Comment:
Just a thought not a must at all, what do you think about extracting those to a separate file, we could add other tour data there as well, in addition, what do you think about merging them to an object list with properties of title and description?

Reply:
this component has a specific digest tour logic, the only thing we can do is move the body to the design system... also the tour library requires you to pass the component like this, so you can't pass the props this way... 
```
const digestTourSteps: Step[] = [
  {...
    tooltipComponent: DigestWorkflowTourTooltip,
}...
```
so if it is specific to the digest tour then I feel it's fine to leave it like it is... also merging to the object won't have here any benefits...